interface/forms/gad7/new.php

```diff
@@ -0,0 +1,219 @@
+<?php
+
+/**
+ * gad-7 form using forms api     new.php    create a new form
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    ruth moulton
+ * @copyright Copyright (c) 2021 ruth moulton ruth@muswell.me.uk
+ *
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+require_once("gad7.inc"); //common strings, other includes etc
+
+use OpenEMR\Common\Csrf\CsrfUtils;    // security module
+use OpenEMR\Core\Header;
+?>
+<html>
+<head>
+    <title><?php echo $string_form_title; ?> </title>
+    <?php Header::setupHeader(); ?>
+</head>
+<body class="body_top">
+<script LANGUAGE="JavaScript">
+var no_qs = 8; // number of questions in the form
+</script>
+<SCRIPT
+  src="<?php echo $rootdir;?>/forms/gad7/gad7_javasrc.js">
+ </script>
+
+<SCRIPT LANGUAGE="JavaScript">
+// stuff that uses embedded php must go here, not in the include javascript file - it must be executed on server side before page is sent to client. included javascript is only executed on the client
+var gad7_score = 0;  //initialise to zero
+function create_q8 ( question, menue){
+ // create the 8th question - the second part is italicised. Only displayed if score > 0
+       var text = document.createTextNode('<?php echo ($str_q8 ); ?>');
+       question.appendChild(text);
+       var new_line = document.createElement("br"); // second part is in italics
+       var ital = document.createElement("i"); // second part is in italics
+       var question_2 = document.createTextNode('<?php echo ( $str_q8_2); ?>');
```

Comment:
```
var question_2 = document.createTextNode(jsAttr(<?php echo js_escape($str_q8_2); ?>));
```

Reply:
is it this section @ruthkonyn that you're removing the quotes?